### PR TITLE
This almost fixes the phantom hand movement issue

### PR DESCRIPTION
**Pull request by omangbaheti**
This fixes the rendering issue by calculating the imaginary centre of the headset. The phantom effect of hand following the headset is almost gone. This might even fix the older implementation of HMD Follower

### CODE REVIEW

File: Assets/Scripts/ViconNexusUnityStream/CustomHWDScript.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@ public class CustomHWDScript : CustomSubjectScript
 
         private OneEuroFilter<Quaternion> rotFilter;
         private OneEuroFilter<Vector3> posFilter;
+        [SerializeField] private float HMDCentreOffset;
```
Comment (ahmed-shariff) [inline Assets/Scripts/ViconNexusUnityStream/CustomHWDScript.cs:32]: This a better place to have this than in the project settings. Consider changing this to a Vector3, then remove the `HMDPositionOffset` and it's references across?

**Review by ahmed-shariff — CHANGES_REQUESTED** [review]
This is intersting. Also, you had set the value to 1.1 something? is that meters?

**Comment by omangbaheti** [comment]
> This is intersting. Also, you had set the value to 1.1 something? is that meters?

So earlier with HMDPositionOffset for just applying the offset not relative to the HMD forward co-ordinates but relative to unity co-ordinate space. So it didnt really make a difference back then. Right now I am setting the values somewhere between (0,0,-100) and (0,0,-250) to get good enough results

**Comment by ahmed-shariff** [comment]
Good catch! Thanx for the fix. I am still not sure why this solves that issue, which is going to keep me up at night 😅

**Pull request merged**